src/bench/checkblock.cpp

```diff
@@ -17,7 +17,7 @@
 static void DeserializeBlockTest(benchmark::Bench& bench)
 {
     CDataStream stream(benchmark::data::block413567, SER_NETWORK, PROTOCOL_VERSION);
-    char a = '\0';
+    uint8_t a{'\0'};
```

Comment:
Nit: for something that looks like an integer type, I find initializing with 0 (`uint8_t a{0};`) more naturally looking than a char literal.